File: bank-service/src/main/java/rs/edu/raf/exceptions/WrongCreditLoanTermException.java
package rs.edu.raf.exceptions;

import org.springframework.http.HttpStatus;

public class WrongCreditLoanTermException extends CustomException{
    public WrongCreditLoanTermException(int minLoanTerm, int maxLoanTerm) {
        super("Credit loan must be between " + minLoanTerm + " and " + maxLoanTerm, HttpStatus.BAD_REQUEST);
    }
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/exception/WrongCreditLoanTermException.kt
package org.example.bankservicekotlin.exception

import org.springframework.http.HttpStatus

data class WrongCreditLoanTermException(val minLoanTerm: Int, val maxLoanTerm: Int) :
    CustomException("Credit loan must be between $minLoanTerm and $maxLoanTerm", HttpStatus.BAD_REQUEST)
